docs/docker-images/redis.md

```diff
@@ -33,21 +32,64 @@ This image is prepared to be used on Lagoon. There are therefore some things alr
+| `LOGLEVEL`           | notice      | Define the level of logs                                                                  |
+| `MAXMEMORY`          | 100mb       | Maximum amount of memory                                                                  |
+| `MAXMEMORYPOLICY`    | allkeys-lru | The policy to use when evicting keys if redis reaches its maximum memory usage            |
+| `REDIS_PASSWORD`     | disabled    | Enables [authentication feature](https://redis.io/topics/security#authentication-feature) |
 
 ## Redis-persistent
 
-The [Lagoon `redis-persistent` Docker image](https://github.com/uselagoon/lagoon-images/blob/main/images/redis-persistent/5.Dockerfile). Based on the [Lagoon `redis` image](./), it is intended for use if the Redis service must be in `persistent` mode \(ie. with a persistent volume where transactions will be saved\).
+Based on the [Lagoon `redis` image](https://github.com/uselagoon/lagoon-images/blob/main/images/redis/5.Dockerfile), the [Lagoon `redis-persistent` Docker image](https://github.com/uselagoon/lagoon-images/blob/main/images/redis-persistent/5.Dockerfile) is intended for use when the Redis service must be utilized in `persistent` mode \(ie. with a persistent volume where keys will be saved to disk\).
 
-It differs from `redis` only for `FLAVOR` environment variable which will use the [respective Redis configurations](https://github.com/uselagoon/lagoon-images/tree/main/images/redis/conf).
+It differs from `redis` only with the `FLAVOR` environment variable, which will use the [respective Redis configuration](https://github.com/uselagoon/lagoon-images/tree/main/images/redis/conf) according to the version of redis in use.
+
+## Troubleshooting
+
+The Lagoon redis images all come pre-loaded with the `redis-cli` command, which allows for querying the redis service for information and setting config values dynamically. To use this utility, you can simply SSH into your redis pod by using the instructions [here] (../using-lagoon-advanced/ssh.md) with `redis` as the `pod` value then run it from the terminal once you've connected.
+
+### Maximum Memory Policy
+
+By default, the Lagoon `redis` images are set to use the `allkeys-lru` policy. This policy will alow **ANY** keys stored in redis to be evicted if/when the redis service hits its `maxmemory` limit according to when the key was least recently used.
+
+For typical installations, this is the ideal configuration, as Drupal may not set a `TTL` value for each key cached in redis. If the `maxmemory-policy` is set to something like `volatile-lru` and Drupal doesn't provide these `TTL` tags, this would result in the redis container filling up, being totally unable to evict **ANY** keys, and ceasing to accept new cache keys at all.
+
+More information on redis' maxmemory policies can be found in redis' [official documentation](https://redis.io/docs/manual/eviction/#eviction-policies).
+
+!!! danger
```

Comment:
follow up from the comment above or we just go with this - I'd just like to make it crystal clear that there can be side-effects changing the maxmemory-policy :)